src/sagemaker_inference/model_server.py

```diff
@@ -129,3 +135,14 @@ def _terminate(signo, frame):
             pass
 
     signal.signal(signal.SIGTERM, _terminate)
+
+
+def _install_requirements():
+    logger.info('installing packages from requirements.txt...')
+    pip_install_cmd = [sys.executable, 'install', '-r', REQUIREMENTS_PATH]
```

Comment:
doesn't `sys.exectuable` return the Python executable? I think you're missing `-m pip`

(alternatively, you could also try importing `pip` and using `pip.main()` instead)

Reply:
Nice catch, I forgot to include pip -m aha

I forgot to link to this guide: https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program

I'm going to go with subprocess call as recommended by the guide.